2nd call, sadface: 9MathError: Special exception caught: 9MathError: a cannot be less than b.
